tests/apollo/test_skvbc_preexecution.py

```diff
@@ -366,6 +366,30 @@ async def test_alternate_f_isolated(self, bft_network, tracker):
             last_block = await tracker.get_last_block_id(read_client)
             assert last_block > start_block
 
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    @verify_linearizability(pre_exec_enabled=True)
+    async def test_conflicting_requests(self, bft_network, tracker):
+        """
+        Launch pre-process conflicting request and make sure that conflicting requests are not committed
+        """
+        bft_network.start_all_replicas()
```

Comment:
once this test is stable, let's add another one where the conflict happens in the presence of _f_ crashed replicas

Reply:
done

Reply:
thanks, almost there - see my comment below...